recipes/itk/all/patches/0001-find-conan-packages.patch

```diff
@@ -0,0 +1,157 @@
+--- Modules/ThirdParty/Eigen3/itk-module-init.cmake
++++ Modules/ThirdParty/Eigen3/itk-module-init.cmake
+@@ -1,3 +1,3 @@
+ if(ITK_USE_SYSTEM_EIGEN)
+-  find_package(Eigen3 REQUIRED CONFIG)
++  find_package(Eigen3 REQUIRED)
```

Comment:
Using `cmake_find_package_multi` would avoid this modification.

Reply:
The build fails when using both `cmake_find_package` and `cmake_find_package_multi`.
It also fails when only using `cmake_find_package_multi`.
This seems to be the easiest option.